### PR TITLE
Update Travis CI

### PR DESCRIPTION
**Pull request by kevina**

### CODE REVIEW

**Comment by kevina** [comment]
Um why is circleci even running, there is no `circle.yml` file?

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,20 @@
 language: go
 sudo: true
 
+os:
+  - linux
+  - osx
```
Comment (Kubuxu) [inline .travis.yml]: There is no need to run osx on repos that don't do any system specific stuff.

**Comment by Kubuxu** [comment]
I've disabled circle.

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]];
-    then make coverage && go vet ./...;
-    else make test;
-    fi
+  - make coverage
+  - go vet ./...
 
 after_success:
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then bash <(curl -s https://codecov.io/bash); fi
+  - curl -s https://codecov.io/bash
```
Comment (Kubuxu) [inline .travis.yml]: You've removed `bash` it is not running.

**Pull request merged**